src/Enzyme.jl

```diff
@@ -780,6 +787,7 @@ end
         adjoint(Duplicated(x, dx), tape)
         return dx
     end
+    mapreduce(LinearAlgebra.adjoint, vcat, rows)
```

Comment:
Kind of hacky, but would it be possible to create the zero init'd matrix, then unsafe_array it at the corresponding offset for each index to avoid making a bunch of temporaries